hint.css

```diff
@@ -17,14 +17,15 @@ html, body {
 .title {
```

Comment:
The index.css uses the `h1 `class name. 
However, the hint.css uses the `.title `class name. 
Please use one of them.